CONTRIBUTING.md

```diff
@@ -43,6 +43,69 @@ GitHub provides additional document on [forking a repository](https://help.githu
 ## Finding contributions to work on
 Looking at the existing issues is a great way to find something to contribute on. As our projects, by default, use the default GitHub issue labels (enhancement/bug/duplicate/help wanted/invalid/question/wontfix), looking at any 'help wanted' issues is a great place to start.
 
+## Getting Started
+### Setup
+To work on this project you need these environmental dependencies:
+-   [NodeJS and NPM](https://nodejs.org/) (latest version of both)
+-   [Git](https://git-scm.com/downloads)
+-   [AWS `git secrets`](https://github.com/awslabs/git-secrets)
+
+Then clone the repository and install the project dependencies with NPM:
+
+```
+git clone git@github.com:aws/amazon-states-language-service.git
+cd amazon-states-language-service
+npm install
+npm test
+```
+
+### Debug
```

Comment:
nit: "Linking with AWS Toolkit for VS Code" would be a more accurate heading. Debugging instructions (setting up a debugger) is mentioned in the `Run` section

Reply:
resolved in #98